### PR TITLE
Changelog updated

### PR DESCRIPTION
**Pull request by david-puglielli**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/816)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,36 +7,36 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
...
-- In Linux and macOS, setlocale() only takes effect if it is invoked before the first connection. Attempting to set the locale after connection will not work
-- Always Encrypted feature, which requires [MS ODBC Driver 17+](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-2017)
-  - only Windows Certificate Store and Azure Key Vault are supported
-  - Issue [#716](https://github.com/Microsoft/msphpsql/issues/716) - With Always Encrypted feature enabled, Named Parameters in Sub Queries are not supported
+- In Linux and macOS, setlocale() only takes effect if it is invoked before the first connection. Attempting to set the locale after connecting will not work
+- Always Encrypted requires [MS ODBC Driver 17+](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-2017)
+  - Only Windows Certificate Store and Azure Key Vault are supported. Custom Keystores are not supported
+  - Issue #716 - With Always Encrypted enabled, named parameters in subqueries are not supported
```
Comment (yitam) [inline CHANGELOG.md]: Suggestion: Custom Keystore provider is not yet supported

**Comment by david-puglielli** [comment]
The full links are not needed in github. Do we still need them in the markdown source?

**Comment by yitam** [comment]
@david-puglielli I see your point. Then never mind. Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,36 +7,36 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
...
+- Issue #577 - Idle Connection Resiliency doesn't work with Column Encryption enabled connections (fixed in MS ODBC Driver 17.1)
+- Issue #678 - Idle Connection Resiliency doesn't work with Connection Pooling (fixed in MS ODBC Driver 17.1)
+- Issue #699 - Binding output parameters fails when the query in the stored procedure returns no data. The test case has been added to the test lab.
+- Issue #705 - Always Encrypted - Retrieving a negative decimal value (edge case) as output parameter causes truncation
+- Issue #706 - Always Encrypted - Cannot insert double with precision and scale (38, 38)
+- Issue #707 - Always Encrypted - Fetching decimals / numerics as output parameters bound to PDO::PARAM_BOOL or PDO::PARAM_INT returns floats, not integers 
+- Issue #735 - Extended the buffer size for PDO::lastInsertId so that data types other than integers can be supported
+- Pull Request #759 - Removed the limitation of binding a binary as inout param as PDO::PARAM_STR with SQLSRV_ENCODING_BINARY
+- Pull Request #775 - Fixed the truncation problem for output params with SQL types specified as SQLSRV_SQLTYPE_DECIMAL or SQLSRV_SQLTYPE_NUMERIC
```
Comment (yitam) [inline CHANGELOG.md]: Just double checked. The links are required for the change log but not on GitHub release. Please put the links back.
